Source/Scene/PointCloud3DTileContent.js

```diff
@@ -511,10 +515,14 @@ define([
         }
 
         var uniformMap = {
-            u_pointSizeAndTilesetTime : function() {
-                scratchPointSizeAndTilesetTime.x = content._pointSize;
-                scratchPointSizeAndTilesetTime.y = content._tileset.timeSinceLoad;
-                return scratchPointSizeAndTilesetTime;
+            u_pointAttenuationMaxSize : function() {
```

Comment:
This is not bundled in `u_pointSizeAndTilesetTimeAndAttenuation` because #5455 overrides just this uniform.

Reply:
Thanks; even though this behavior might seem redundant; it makes the uniform map easier to modify from future stages